frontend/src/pages/Home/Home.tsx

```diff
@@ -1,41 +1,33 @@
-import Pokemon from "components/Pokemon/Pokemon"
 import styles from "./Home.module.css"
 import { useEffect, useState } from "react"
-
-type PokemonList = {
-  pokemonList: Pokemon[]
-  filterValue: string
-}
-
-type Pokemon = {
-  name: string
-  id: number
-  weight: number
-  height: number
-}
-
-function filterPokemonsByName(pokemons: Pokemon[], filteredName: string) {
-  return pokemons.filter(({ name }) => name.toLowerCase().includes(filteredName))
-}
+import Loader from "components/Loader/Loader"
+import Error from "components/Error/Error"
+import PokemonList from "components/PokemonList/PokemonList"
 
 const fetchPokemons = async (url: string) => {
   try {
     const response = await fetch(url, { headers: { accept: "application/json" } })
     const jsonResponse = await response.json()
     return jsonResponse
   } catch (error) {
-    console.log(error)
-    return []
+    throw error
   }
 }
```

Comment:
- <!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
    
    **🎯 suggestion (non-blocking):** <subject>This code regards the integration with the Pokemon API. It could be separated in a dedicated files `/PokemonList/data_integration/fetchPokemons.ts`
    
    <discussion>